mentionnutil/extractor_test.go

```diff
@@ -0,0 +1,37 @@
+package mentionnutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractMentions(t *testing.T) {
+	for _, tt := range []struct {
+		name   string
+		input  string
+		expect []string
+	}{
+		{
+			name: "empty",
+		},
+		{
+			name:  "no mentions",
+			input: "foo bar",
+		},
+		{
+			name:   "multi mentions",
+			input:  "foo bar @buz. @taz_ @dot.name hello @buz @with_1",
+			expect: []string{"buz", "taz_", "dot.name", "with_1"},
+		},
+		{
+			name:   "youtube url",
+			input:  "foo bar @buz. https://www.youtube.com/@SalonViking test",
```

Comment:
The bad thing of this it's this catch also handle from tiktok or instagram urls but we didn't know at the end at which social media it's linked. This could lead to some confusion. Maybe must be more restrictive ?